### PR TITLE
Made additions to use localhost of VM

### PR DESCRIPTION
**Pull request by odrodrig**
- Added commands to access localhost of VM
- Added curl commands for health endpoints
- Added additional clarification

### CODE REVIEW

File: workshop/exercise-2/README.md [hunk 1]
```diff
@@ -158,19 +163,46 @@ It's possible to run this application on your workstation immediately.
 appsody run
 ```
 
-You can see the output of the application by navigating to `http://localhost:3000` in a browser window.
+You can see the output of the application by opening another browser tab to the terminal environment and running the following:
 
-Appsody builds a containerized version of the application for you and runs it in Docker. You can enter `http://localhost:3000` in a browser to see the default endpoint served by the application.
+```bash
+VM_IP=$(hostname -I | awk '{ print $2 }')
+echo $VM_IP:3000
```
Comment (timroster) [inline workshop/exercise-2/README.md]: prepend http:// to the echo

File: workshop/exercise-2/README.md [hunk 2]
```diff
@@ -220,10 +254,18 @@ The resulting directory structure of your Appsody project should look like this:
...
 
-Watch for the container to restart and then refresh your browser again. You will see a form appear.
+This is what the frontend looks like:
 
 ![Sample web form](images/screenshot.png)
 
+You can see the application by running the following:
+
+```bash
+echo $VM_IP:3000/quote
```
Comment (timroster) [inline workshop/exercise-2/README.md]: also add http:// to command

**Review by timroster — CHANGES_REQUESTED** [review]
small updates needed for example as noted

File: workshop/exercise-2/README.md (hunk 1 above)
Comment (odrodrig) [reply workshop/exercise-2/README.md]: fixed in [this commit](https://github.com/IBM/cloudpakforapps-workshop/pull/94/commits/11503d50fa71c8d0ddbf5c3b49e53811eab5918a)

File: workshop/exercise-2/README.md (hunk 2 above)
Comment (odrodrig) [reply workshop/exercise-2/README.md]: fixed in [this commit ](https://github.com/IBM/cloudpakforapps-workshop/pull/94/commits/11503d50fa71c8d0ddbf5c3b49e53811eab5918a)

**Pull request merged**